lib/commands/startTest.js

```diff
@@ -29,9 +29,6 @@ async function startTest(clientTestId, {name, description} = {}) {
 		description,
 	};
 
-	// validate content json
-	validate(validators.START_TEST, content, invalidInputMessage(startTest.name, 'Start test data'));
-
 	// authorize
 	const authedContent = await authContext.authorizeWs({
 		...content,
```

Comment:
I think you need remove `...content` for prevent sending clientTestId, name, description to server.

Reply:
removed